src/lib/init/prepare-template.js

```diff
@@ -5,25 +5,18 @@ export async function prepareTemplate(basename, appName) {
   const arappPath = path.resolve(basename, 'arapp.json')
   const arapp = await fs.readJson(arappPath)
 
-  // TODO remove once the old arapp.json is no longer supported
-  if (!arapp.environments) {
-    arapp.environments = {}
-  }
-
-  const props = {
-    network: 'development',
-    appName: appName,
-  }
-
-  if (arapp.environments.default) {
-    Object.assign(arapp.environments.default, props)
-  } else {
-    arapp.environments.default = props
-  }
-
-  // remove old arapp.json props
-  delete arapp.appName
-  delete arapp.version
+  const defaultEnv = arapp.environments.default
+  const stagingEnv = arapp.environments.staging
+  const productionEnv = arapp.environments.production
+
+  defaultEnv.appName = appName
+  Object.assign(arapp.environments.default, defaultEnv)
+
+  stagingEnv.appName = stagingEnv.appName.replace(/^app/, appName)
```

Comment:
To avoid `stagingEnv.appName = 'aragon-app.aragonpm.eth.open.aragonpm.eth'`

we could have:
```diff
- stagingEnv.appName = stagingEnv.appName.replace(/^app/, appName)
+ stagingEnv.appName = stagingEnv.appName.replace(/^app/, basename)
```

this will work for the CLI, but break the test, because in the test we are passing a path: `./.tmp/aragon-app`, and not the basename.

I think what we can do instead is some parsing in `prepareTemplate` (get the basename from appName) and rename the param to `path`:

```js
function prepareTemplate(path, appName) {
   const basename = name.split('.')[0]
   //...
}
```

Reply:
Yes, you are right. This is better